Plugin/CppApi/source/CoordinateConversion/CoordinateConversionController.cpp

```diff
@@ -276,8 +276,32 @@ bool CoordinateConversionController::isFormat(CoordinateConversionOptions *optio
 
 void CoordinateConversionController::setGeoView(QObject* geoView)
 {
-  if (setGeoViewInternal(dynamic_cast<GeoView*>(geoView)))
-    connect(geoView, SIGNAL(mouseClicked(QMouseEvent&)), this, SLOT(onMouseClicked(QMouseEvent&)));
+  if (!geoView)
+    return;
+
+  if (strcmp(geoView->metaObject()->className(), MapQuickView::staticMetaObject.className()) == 0)
```

Comment:
std::strcmp?

Reply:
include <cstring>? I'm not sure if this function will work on all platforms if the include is missing